src/main/java/com/zufar/onlinestore/cart/endpoint/CartEndpoint.java

```diff
@@ -48,4 +50,16 @@ public ResponseEntity<ShoppingSessionDto> updateProductsQuantityInShoppingSessio
                 .body(shoppingSessionDto);
     }
 
+    @DeleteMapping
+    public ResponseEntity<ShoppingSessionDto> deleteItemInShoppingSession (
+            @RequestBody @Valid final RemoveItemFromShoppingSessionRequest request) {
+        log.info("Received the request to delete the shopping session items with ids: {}.",
+                request.shoppingSessionItemId());
+        ShoppingSessionDto shoppingSessionDto = cartApi.removeItemFromShoppingSession(request);
+        log.info("The shopping session items with ids = {} was deleted.",
```

Comment:
The shopping session items with ids = {} were deleted

Reply:
fixed

Reply:
fixed